modules/widgets/facebook-likebox.php

```diff
@@ -76,9 +76,21 @@ function widget( $args, $instance ) {
 		$title    = apply_filters( 'widget_title', $instance['title'] );
 		$page_url = set_url_scheme( $like_args['href'], 'https' );
 
-		$like_args['show_faces'] = (bool) $like_args['show_faces'] ? 'true' : 'false';
-		$like_args['stream']     = (bool) $like_args['stream'] ? 'timeline' : 'false';
-		$like_args['cover']      = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['show_faces']   = (bool) $like_args['show_faces'] ? 'true' : 'false';
+		$like_args['stream']       = (bool) $like_args['stream'] ? 'timeline' : 'false';
+		$like_args['cover']        = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['small_header'] = (bool) $like_args['small_header'] ? 'true' : 'false';
```

Comment:
What's the current default value for this header? If I update Jetpack, will my header change unless I go to the widget options and toggle this new option?

Reply:
Fixed this by adding a value of `true` to `small_header` to `get_default_args` function. I think that's what you wanted here but let me know if I'm wrong 😅

Reply:
I think it should default to `false`, since that's the default value in the Page Plugin. The idea is for the widget to look the same when folks will update Jetpack, so the large header has to stay until you decide to have a smaller header instead.